python/magma_access_gateway_installer/agw_installer.py

```diff
@@ -177,8 +151,8 @@ def preconfigure_wireshark_suid_property():
     def install_magma_agw(self):
         """Installs Magma AGW's deb packages from the private apt repository."""
         logger.info("Installing Magma Access Gateway...")
-        dpkg_options = ["force-overwrite", "force-confold", "force-confdef"]
-        self._install_apt_package("magma", dpkg_options)
+        for magma_package in self.MAGMA_PACKAGES:
+            self._install_apt_package(magma_package)
```

Comment:
I think this would be a great opportunity to unroll this loop and install all packages in the same `apt` command, this would speed up the installation process significantly.

Reply:
Fixed